### PR TITLE
Module initial release v1.0.0

### PR DESCRIPTION
**Pull request by stereomon**
- Developer(s): @stereomon

- Peer Reviewer:

- Ticket: https://spryker.atlassian.net/browse/CORE-4452

- Academy PR: ACADEMY_URL_HERE

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department
- [ ] All new or heavily changed classes get an "A" rating (Scrutinizer), except Facades, Factories, QueryContainers and Tests

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   NewRelic              | major (new)           |                       |

#### Release Notes

New Module which brings a MonitoringExtension for Spryker's Monitoring Module.

-----------------------------------------

#### Module NewRelic

_**Mini-Major:** Added functionality with incompatible API changes_

Def of done (by responsible developer):
- [ ] API methods are defined and specified [API-doc](https://academy.spryker.com/understanding_spryker/definition_api.html).
- [ ] New and changed facade methods covered by functional tests. / Has nothing to test.
- [ ] New and changed complex business logic is covered by unit or integration tests. / Has nothing to test.
- [ ] Cleanups are resolved. / No relevant cleanups. [cleanups](https://release.spryker.com/module-cleanups).
- [ ] Migration guide provided.

### Initial Release

- Added NewRelic MonitoringExtension Plugin.

### CODE REVIEW

File: src/SprykerEco/Shared/NewRelic/NewRelicConstants.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
```
Comment (dereuromark) [inline src/SprykerEco/Shared/NewRelic/NewRelicConstants.php]: Is the .license not working here?

File: src/SprykerEco/Shared/NewRelic/NewRelicConstants.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+namespace SprykerEco\Shared\NewRelic;
+
+interface NewRelicConstants
```
Comment (dereuromark) [inline src/SprykerEco/Shared/NewRelic/NewRelicConstants.php]: This should use the new standards NewRelicEnv (similar to RabbitMqEnv) etc IMO.

File: src/SprykerEco/Shared/NewRelic/NewRelicConstants.php (hunk 1 above)
Comment (stereomon) [reply src/SprykerEco/Shared/NewRelic/NewRelicConstants.php]: Looks like code sniffer is broken here.

File: .gitignore [hunk 3]
```diff
@@ -26,3 +26,4 @@
 /node_modules/
 /tests/_output/*
 !/tests/_output/.gitkeep
+vendor
```
Comment (dereuromark) [inline .gitignore:29]: Please add also composer lock as always for any non project repo.

File: composer.json [hunk 4]
```diff
@@ -4,7 +4,14 @@
...
     "php": ">=7.1",
-    "spryker/monitoring-extension": "^1.0.0"
+    "spryker/monitoring-extension": "^1.0.0",
+    "spryker/kernel": "^3.0.0",
+    "psr/http-message": "^1.0.0",
+    "guzzlehttp/guzzle": "^6.0.0"
+  },
+  "require-dev": {
+    "codeception/codeception": "^2.0.0",
+    "phpstan/phpstan": "^0.9.2"
```
Comment (dereuromark) [inline composer.json]: ideally 0.10. and level 3-5

File: .license [hunk 5]
```diff
@@ -0,0 +1,4 @@
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
```
Comment (dereuromark) [inline .license]: This is the wrong text for our MIT license.
Please see https://github.com/spryker/code-sniffer/blob/master/.license

**Review by alex-galych — APPROVED** [review]

**Pull request merged**